backtrace-library/src/main/java/backtraceio/library/services/BacktraceDatabaseContext.java

```diff
@@ -372,4 +371,21 @@ private BacktraceDatabaseRecord getRecordFromCache(boolean reverse) {
         }
         return null;
     }
+
+    private void addToFirstBatch(BacktraceDatabaseRecord backtraceDatabaseRecord) {
+        final int firstBatch = 0;
+
+        if (this.batchRetry.isEmpty()) {
```

Comment:
how is this even possible if in the custructor, you're making sure at least one batch is being created? L90

Reply:
it's IDE thing - IntelliJ is not sure if it's initialized because `get` method has annotation `Nullable` but I think it's not that bad to have this extra check which I've introduce if in the future someone by mistake will change this code a bit

Reply:
+ before my change it was doable when someone will pass e.g. _retryNumber = -1